openapi-cli/src/main/java/org/ballerinalang/openapi/cmd/OpenApiCmd.java

```diff
@@ -156,8 +159,14 @@ private void ballerinaToOpenApi(String fileName) throws IOException {
 //        Path resourcePath = getRelativePath(new File(balFilePath.toString()), this.targetOutputPath.toString());
         //ballerina openapi -i service.bal --serviceName serviceName --module exampleModul -o ./
         // Check service name it is mandatory
+        boolean isJson;
+        if (generatedFileType) {
```

Comment:
You can directly assign isJson to generatedFileTypes right ?

Reply:
yeah, I will directly assign this value.